index.html

```diff
@@ -10,224 +10,293 @@
   <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
 </head>
 <body>
-  <section class="main-section">
+  <main class="main-section">
     <header class="header-section">
-      <p class="logo"><b> My Logo</b> </p> 
+      <p class="logo"><b>My Logo</b> </p> 
 
 <nav class="bttn">
-      <span class="line"></span>
-      <span class="line"></span>
-      <span class="line"></span>
+    <ul class="nav-list">
+        <li><a href="#portfolio">Portfolio</a></li>
+        <li><a href="#about">About</a></li>
+        <li><a href="#contact">contact</a></li>
+    </ul>
 </nav>
   </header>
   <br> <br>
-  <section  >
-    <img  class="background-image" src="shapes/i-bttn/background-shapes/header-shapes mobile.png" alt="">
-     
-      <div>   <h1 class="tex">I’m Trevor <br> Glad to see you!</h1>
-     <p class="head-section">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-  </div>
-     <div>
- <h2 class="lets"> LET'S CONNECT </h2> 
- </div>
- <br>
- <div>
-                 <ul class="icons">
+   
+    <section class="shapes" > 
+    <div>   <h1 class="tex">I’m Trevor <br> Glad to see you!</h1>
+   <p class="head-section">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+</div>
+   <div>
+
+<h2 class="lets"> LET'S CONNECT </h2> 
+
+</div>
+<br>
+<div>
+               <ul class="icons">
+                <a href="" class="icon">
+                <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20" fill="none">
+                    <path d="M19.7587 0.394423C19.4253 -0.00424693 18.8587 -0.103914 18.4253 0.195088L16.492 1.52399C15.592 0.560536 14.3253 0.0289756 12.9587 0.0289756C10.2587 0.0289756 8.02532 2.22166 8.02532 4.94591C8.02532 5.34458 8.05865 5.71002 8.15865 6.07547C5.82532 5.57713 4.42532 3.88279 3.89198 0.826316C3.82532 0.427646 3.52532 0.128643 3.12532 0.0289756C2.75865 -0.070692 2.35865 0.0954206 2.12532 0.394423C2.05865 0.460868 -3.44135 8.10205 4.52532 14.5804C3.75865 16.2083 1.45865 17.6037 0.558651 18.0688C0.125318 18.2681 -0.0746825 18.7332 0.0253175 19.1984C0.125318 19.6635 0.525318 19.9625 0.991984 19.9625H2.72532C12.8587 19.9625 17.4587 14.9459 17.5587 3.71667L19.6587 1.72332C20.0587 1.3911 20.092 0.793093 19.7587 0.394423Z" fill="#505F79"/>
```

Comment:
- [ ] You are almost done with this headline part 👌 .  There are a few changes to apply to have it completed 💯 .
  - I suggest that you make the background image cover 100% width of the headline. This is because we want to ensure that you match the Figma design.
  - I suggest reducing the space on top and bottom of `LET'S CONNECT`. By applying this, your project will look professional and match the design provided to follow 🚶‍♂️ .
  - If you notice from the Figma design, there is [`border-bottom-left-radius`](https://developer.mozilla.org/en-US/docs/Web/CSS/border-bottom-left-radius#:~:text=The%20border%2Dbottom%2Dleft%2D,the%20curvature%20of%20the%20corner.). Kindly ensure to apply this ☺️ .
  - Kindly ensure that the font matches the Figma design for the project to look professional.

- Your design 😎 
![headlin](https://user-images.githubusercontent.com/99394374/213878306-57838025-affa-4a3a-9592-e70222391979.PNG)

- Figma design 😎 
![fig headl](https://user-images.githubusercontent.com/99394374/213878322-44880204-1108-4e92-86cb-a125f6b99fe2.PNG)